apps/homepage/src/app/(with-header)/mypage/admin/attendance/_containers/TabContainer.tsx

```
@@ -12,7 +12,13 @@ import {
   ATTENDANCE_STATUS_CONFIG,
   ATTENDANCE_STATUS_OPTION,
   AttendanceStatusKey,
-} from '@/constants/admin/admin-attendance';
+} from '@/constants/admin/admin';
+import {StatusChip} from '@repo/ui/components/chip/StatusChip';
```

Biome (2.4.4)

[error] Shouldn't redeclare 'StatusChip'. Consider to delete it or rename it.

(lint/suspicious/noRedeclare)